From 5d6d09c2338ae62796c2a326340e81320faf3593 Mon Sep 17 00:00:00 2001
From: Vita Preskovsky <vitap@ti.com>
Date: Wed, 19 Sep 2012 13:05:18 +0200
Subject: [PATCH] obexd: make OPP push timeout longer

Current OPP push timeout is not long enough for user to confirm on OPP server
to receive file. As a result, OPP client exists with error.


Signed-off-by: Vita Preskovsky <vitap@ti.com>
---
 plugins/bluetooth.c |    2 +-
 src/manager.c       |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/plugins/bluetooth.c b/plugins/bluetooth.c
index 851daa2..204ccf5 100644
--- a/plugins/bluetooth.c
+++ b/plugins/bluetooth.c
@@ -46,7 +46,7 @@
 #define BT_RX_MTU 32767
 #define BT_TX_MTU 32767
 
-#define TIMEOUT 60*1000 /* Timeout for user response (miliseconds) */
+#define TIMEOUT 180*1000 /* Timeout for user response (miliseconds) */
 
 struct pending_request {
 	DBusPendingCall *call;
diff --git a/src/manager.c b/src/manager.c
index 80140b6..e64a433 100644
--- a/src/manager.c
+++ b/src/manager.c
@@ -52,7 +52,7 @@
 #define TRANSFER_INTERFACE OPENOBEX_SERVICE ".Transfer"
 #define SESSION_INTERFACE OPENOBEX_SERVICE ".Session"
 
-#define TIMEOUT 60*1000 /* Timeout for user response (miliseconds) */
+#define TIMEOUT 180*1000 /* Timeout for user response (miliseconds) */
 
 struct agent {
 	char *bus_name;
-- 
1.7.9.5

